File: ProvaGFT/src/exercicio01/Supervisor.java
package exercicio01;

public class Supervisor extends Funcionario{

	public Supervisor(String nome, int idade, double salario) {
		super(nome, idade, salario);
		
	}
	
	@Override
	public void Bonificacao() {
		this.setSalario(this.getSalario()+5000.00);	
		System.out.println(this.getNome() + ", com o salário de " + this.getSalario());
	}
	
	

}

File: Prova/Supervisor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Prova
{
    class Supervisor : Funcionarios
    {
        public Supervisor(string nome, int idade, double salario) : base(nome, idade, salario)
        {
        }

        public void Bonificacao()
        {
            Salario = Salario + 5000.00;
            Console.WriteLine("O salário do " + Nome + " é de " + Salario + " reais.");
        }

    }
          



}
